internal/dataplane/sendconfig/kong.go

```diff
@@ -60,24 +89,10 @@ func New(
 		})
 	}
 	if err := errg.Wait(); err != nil {
```

Comment:
At this point this could be made simply: `return errg.Wait()`

Reply:
Done